### PR TITLE
Extract new strings (2024-11-20)

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -2996,6 +2996,10 @@
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="15.4" build-num="15F31d"/>
     </header>
     <body>
+      <trans-unit id="Bookmarks" xml:space="preserve">
+        <source>Bookmarks</source>
+        <note>The title of the folder that contains mobile bookmarks. This should match bookmarks.folder.mobile.label on Android.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3001]: @dataports: Seems like you landed this string. Could you please update the comment, by removing "This should match bookmarks.folder.mobile.label on Android."? Localizers don’t refer to connections among strings like this.
Also, side-note (but not blocking): the string ID used here is simply "Bookmarks" -> would be more specific if it was updated to something like used in previous string IDs (example of a previous string ID: "Bookmarks.Menu.BookmarksTopLevelTitle.v131" which is more reflective

**Review by Delphine — CHANGES_REQUESTED** [review]

File: templates/firefox-ios.xliff (hunk 1 above)
Comment (dataports) [reply templates/firefox-ios.xliff:3001]: Hi there, opened a PR here https://github.com/mozilla-mobile/firefox-ios/pull/23250

Comment (dataports) [reply templates/firefox-ios.xliff:3001]: RE: your side note. 

The key for the localization string in Swift in this case is being used as the string to translate, so I can't change it without refactoring this area to use the value for the string, and I'm unsure of the consequences for previous versions, so I won't implement a different stringID in this specific case, but I will keep it in mind for next time.

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -3175,6 +3179,22 @@ Private Tab</source>
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="15.4" build-num="15F31d"/>
     </header>
     <body>
+      <trans-unit id="Alerts.AddToCalendar.Body.v134" xml:space="preserve">
+        <source>%@ is asking to download a file and add an event to your calendar.</source>
+        <note>When tapping on a link, on a website in order to download a file and that file is a calendar file, an alert comes up asking to confirm if you want to add the event to the device calendar. This is the body message for the alert. %@ is the name/domain of the website, for example 'google.com'</note>
+      </trans-unit>
+      <trans-unit id="Alerts.AddToCalendar.BodyDefaultDomain.v134" xml:space="preserve">
+        <source>This site</source>
```
Comment (flodolo) [inline templates/firefox-ios.xliff:3187]: @adudenamedruby @dicarobinho
Flagged this in https://github.com/mozilla-mobile/firefox-ios/pull/23232#issuecomment-2488960898

**Review by flodolo — APPROVED** [review]

**Pull request merged**